docs/apis/embeds.md

```diff
@@ -0,0 +1,169 @@
+# Embeddables
+
+**Embeddables are content blocks that can be placed in-between other DOM elements, while keeping a native look. Embeddables are shown to your end-users when they meet all of the predefined matching criteria, including:**
```

Comment:
I mostly followed the same structure from Tours. Biggest change was this intro + removing Tours references